nuxeo-opencmis-tests/src/test/java/org/nuxeo/ecm/core/opencmis/impl/TestCmisBinding.java

```diff
@@ -2264,6 +2266,48 @@ public void testQueryContains() throws Exception {
         }
     }
 
+    /**
+     * Test the relax mode having multiple {@code CONTAINS()}s in CMISQL. The relax mode does not follow the CMIS
+     * specification 1.1 where at most one {@code CONTAINS()} function MUST be included in a single query statement
+     * (Spec 2.1.14.2.4.4). This relax mode only work for CMISQL query having no {@code JOIN} predicate.
+     *
+     * @throws Exception
+     * @see https://jira.nuxeo.com/browse/NXP-19858
+     */
+    @Test
+    @LocalDeploy("org.nuxeo.ecm.core.opencmis.tests.tests:OSGI-INF/test-unfollow-cmis-spec.xml")
+    public void testQueryMultiContains() throws Exception {
+
+        if (supportsJoins()) {
+            return;
+        }
+
+        final ConfigurationService configService = Framework.getService(ConfigurationService.class);
```

Comment:
I added `final` to ensure that there's no reference change during the test. Could you explain more about the deletion, maybe it's evident that it won't be changed? I just want to learn more about the good practices.